src/qt/pivx/sendconfirmdialog.cpp

```diff
@@ -78,6 +80,7 @@ TxDetailDialog::TxDetailDialog(QWidget *parent, bool isConfirmDialog, QString wa
     connect(ui->btnEsc, SIGNAL(clicked()), this, SLOT(closeDialog()));
     connect(ui->pushInputs, SIGNAL(clicked()), this, SLOT(onInputsClicked()));
     connect(ui->pushOutputs, SIGNAL(clicked()), this, SLOT(onOutputsClicked()));
+    grabKeyboard();
```

Comment:
hmm, this will obtain the keyboard fully for the parent widget and not for the widget's child and requires to add code in the destructor (which we should avoid as it could lead to future errors).

Would suggest to use `installEventFilter` like what we are doing in the `AskPassphraseDialog` to catch the keyboard event.